pkg/reconciler/v1alpha1/service/service_test.go

```diff
@@ -507,6 +507,69 @@ func TestReconcile(t *testing.T) {
 		WantEvents: []string{
 			Eventf(corev1.EventTypeNormal, "Updated", "Updated Service %q", "route-fails"),
 		},
+	}, {
+		Name:    "runLatest - not owned config exists",
+		WantErr: true,
+		Objects: []runtime.Object{
+			svc("run-latest", "foo", WithRunLatestRollout),
+			config("run-latest", "foo", WithRunLatestRollout, WithConfigOwnersRemoved),
+		},
+		Key: "foo/run-latest",
+		WantStatusUpdates: []clientgotesting.UpdateActionImpl{{
+			Object: svc("run-latest", "foo", WithRunLatestRollout,
+				// The first reconciliation will initialize the status conditions.
+				WithInitSvcConditions, MarkConfigurationNotOwned),
+		}},
+	}, {
+		Name:    "runLatest - not owned route exists",
+		WantErr: true,
+		Objects: []runtime.Object{
+			svc("run-latest", "foo", WithRunLatestRollout),
+			config("run-latest", "foo", WithRunLatestRollout),
+			route("run-latest", "foo", WithRunLatestRollout, WithRouteOwnersRemoved),
+		},
+		Key: "foo/run-latest",
+		WantStatusUpdates: []clientgotesting.UpdateActionImpl{{
+			Object: svc("run-latest", "foo", WithRunLatestRollout,
+				// The first reconciliation will initialize the status conditions.
+				WithInitSvcConditions, MarkRouteNotOwned),
+		}},
+	}, {
+		Name: "runLatest - route and config ready, propagate ready",
```

Comment:
change the test name, I think such name  already exists (I wrote it, iirc :-))

Reply:
Good catch, fixed.